### DISCUSSION TITLE
Can't generate graph

### DISCUSSION THREAD

**Discussion by pathos110304** [nicolargo/glances#2323] [Q&A] [answered]
Hello, 

When 'g' key is pressed, the gui shows "generating graph in /tmp" but the graph file is not in /tmp. Only see several log files and it doesn't say anything about graph error. Tried "--export-graph-path /Path" as well but no luck.  What am i missing? where do i see this graph file from? Please help. Thank you.

**Comment by nicolargo** [comment, marked as answer]
Hi @pathos110304 

in order to enable the graph export, you need to run Glances with the following command line:

    glances --export graph

i've open an issue to improve the docs/user message.
